incubator/mysqlha/templates/statefulset.yaml

```diff
@@ -149,6 +155,13 @@ spec:
             secretKeyRef:
               name: {{ template "fullname" . }}
               key: mysql-replication-password
+        - name: MYSQL_METRICS_USER
```

Comment:
Please add ``` {{- if .Values.metrics.enabled }} ``` here, ending on line 164 otherwise the deployment will fail because the secret will not exist.